packages/atomic/src/components/search/result-template-components/quickview-iframe/quickview-iframe.spec.ts

```diff
@@ -38,16 +30,13 @@ describe('QuickviewIframe (Stencil)', () => {
     src?: string;
     logger?: SearchEngine['logger'];
   }): Promise<HTMLIFrameElement> => {
-    // Call the actual Stencil functional component with required parameters
-    const vnode = QuickviewIframe(
-      props,
-      [], // children
-      // biome-ignore lint/suspicious/noExplicitAny: Stencil FunctionalComponent requires utils parameter but it's not used
-      {} as any
-    ) as VNode;
-
-    const element = await renderStencilVNode(vnode, container);
-    return element as HTMLIFrameElement;
+    render(html`${renderQuickviewIframe({props})}`, container);
+
+    // Twice because renderQuickviewIframe has 2 nested async updates
+    await new Promise((resolve) => setTimeout(resolve));
+    await new Promise((resolve) => setTimeout(resolve));
```

Comment:
I tried using `requestAnimationFrame` here, but some tests still required 2 calls. Calling this twice is the fastest _and_ most reliable.